sdk/python/lib/test/langhost/util.py

```diff
@@ -94,16 +94,17 @@ def RegisterResource(self, request, context):
         delete_before_replace = request.deleteBeforeReplace
         ignore_changes = sorted(list(request.ignoreChanges))
         version = request.version
+        import_ = request.importId
 
         property_dependencies = {}
         for key, value in request.propertyDependencies.items():
             property_dependencies[key] = sorted(list(value.urns))
 
         outs = {}
         if type_ != "pulumi:pulumi:Stack":
-            outs = self.langhost_test.register_resource(
-                context, self.dryrun, type_, name, props, deps, parent, custom, protect, provider,
-                property_dependencies, delete_before_replace, ignore_changes, version)
+            rrsig = signature(self.langhost_test.register_resource)
+            args = [context, self.dryrun, type_, name, props, deps, parent, custom, protect, provider, property_dependencies, delete_before_replace, ignore_changes, version, import_]
+            outs = self.langhost_test.register_resource(*args[0:len(rrsig.parameters)])
```

Comment:
Just wondering what caused you to make these changes (vs just appending yet another argument)

Reply:
> Just wondering what caused you to make these changes (vs just appending yet another argument)

Updating every single Python test each time we make a change is both tedious and inconsistent with the flow for Node.